modules/api/src/main/java/org/eclipse/xpanse/api/ServiceDeployV2Api.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.Collections;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.xpanse.modules.deployment.DeployServiceV2;
+import org.eclipse.xpanse.modules.models.response.Response;
+import org.eclipse.xpanse.modules.models.service.deploy.CreateRequest;
+import org.eclipse.xpanse.modules.orchestrator.deployment.DeployTask;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * REST interface methods for processing Service.
+ */
+@Slf4j
+@RestController
+@Profile("remote-deploy")
+@RequestMapping("/xpanse")
+@CrossOrigin
+public class ServiceDeployV2Api {
```

Comment:
we need to use the same existing APIs. and based on the profile.. we must be able to switch between different terraform deployment. 
Please note that even this profile based selection is also only temporary. Once we have terraform-boot fully working, we will move to that completely and delete all terraform execution within xpanse runtime. 
So its fine even if you can think of any temporary/manual solution to start the xpanse runtime to use terraform-boot for deployment